Features/Context/SubContext/CommonActions.php

```diff
@@ -293,7 +293,14 @@ public function iSeeElementFile($element, $file)
      */
     public function iShouldBeOnTheDashboard()
     {
-        $this->waitWhileLoading();
-        $this->assertSession()->elementExists('css', 'div.ez-view-dashboardview');
+        $this->assertSession()->elementExists('css', '.ez-view-dashboardview');
+    }
+
+    /**
+     * @Given /^I am redirected to a location view$/
+     */
+    public function iHaveBeenRedirectedToALocationView()
+    {
+        $this->assertSession()->elementExists('css', '.ez-mainviews .ez-locationview-content');
```

Comment:
`.ez-locationview-content` is more an implementation detail, you should instead look for `.ez-mainviews .ez-view-locationviewview`.